README.md

```diff
@@ -33,54 +45,56 @@ By default, the prefix is `/lambda/mipsSecret`, resulting the following required
+| CodesToOmit        | CodesToOmit          | List of numeric codes to remove from output.       |
+| NoProgramCode      | NoProgramCode        | Numeric code to use for "No Program" entry.        |
+| OtherCode          | OtherCode            | Numeric code to use for "Other" entry.             |
 
 ### Query String Parameters
 
-A couple query-string parameters are available to configure response output.
+Several query-string parameters are available for either endpoint to configure
+response output.
 
-An `enable_code_filter` parameter is available for the `/accounts` endpoint to
-optionally deduplicate codes, removing inactive codes and `CodesToOmit`.
-Defining any non-false value for this parameter will enable it.
+| Query String Parameter | Allowed Values                        |
+|------------------------|---------------------------------------|
+| show_other_code        | "on" or "yes" or "true"               |
+| hide_no_program_code   | "on" or "yes" or "true"               |
+| show_inactive_codes    | "on" or "yes" or "true"               |
+| only_accounts          | Comma-separated list of numeric codes |
+| limit                  | Integer                               |
 
-A `disable_inactive_filter` parameter is available to modify the behavior of
-`enable_code_filter`. If this parameter is set to any non-false value, inactive
-codes are no longer removed. Codes are still deduplicated and `CodesToOmit` is
-respected. This parameter has no effect if `enable_code_filter` is not enabled.
+An `show_other_code` parameter is available to optionally include an "Other"
+entry in the output with a value from the `OtherCode` parameter. Defining any
+non-false value for this parameter will include the code.
 
-An `enable_other_code` parameter is available for either endpoint to optionally
-include an "Other" entry in the output with a value from the `OtherCode`
-parameter. Defining any non-false value for this parameter will enable it.
+A `hide_no_program_code` parameter is available to prevent the default addition
+of a "No Program" entry, otherwise it will be added with the value from the
+`NoProgramCode` parameter. Defining any non-false value for this parameter will
+prevent the addition of a "No Program" entry.
 
-A `disable_no_program_code` parameter is available for either endpoint to optionally
-exclude a "No Program" entry from the output, otherwise it will be injected
-with a value from the `NoProgramCode` parameter.
-Defining any non-false value for this parameter will enable it.
+A `show_inactive_codes` parameter is available to prevent the default removal of
+inactive codes. If this parameter is set to any non-false value, inactive codes
+will be included in the output.
 
-A `limit` parameter is available for either endpoint to restrict the number of
-items returned. This value must be a positive integer, a value of zero
-disables the parameter.
+An `only_accounts` parameter is available to ignore all entries from the chart
```

Comment:
it's not real clear what the default behavior is when `only_accounts` contains inactive codes.  will the code still be hidden by default and you still need to set `show_intactive_codes=true` to make it visible?  i think it would be good to make it clear and probably have a unit test for it.